src/app/rpmostree-libbuiltin.cxx

```diff
@@ -83,18 +83,21 @@ rpmostree_has_new_default_deployment (RPMOSTreeOS *os_proxy,
   return !g_str_equal (previous_id, new_id);
 }
 
+namespace rpmostreecxx {
+
 /* Print the diff between the booted and pending deployments */
-gboolean
-rpmostree_print_treepkg_diff_from_sysroot_path (const gchar   *sysroot_path,
-                                                RpmOstreeDiffPrintFormat format,
-                                                guint          max_key_len,
-                                                GCancellable  *cancellable,
-                                                GError       **error)
+void
+print_treepkg_diff_from_sysroot_path (rust::Str      sysroot_path,
```

Comment:
Could split this as a prep commit?

Reply:
Yeah, lots of things to split out of this!